.github/workflows/verify.yaml

```diff
@@ -22,7 +22,6 @@ jobs:
 
       # lets run Checkstyle explicitly (as opposed to within gradle) due to better reporting capabilities
       - name: Run Checkstyle
-        if: github.event_name == 'pull_request'
```

Comment:
As `reporter: 'github-pr-check'` so this will only work on PRs and not on commits. And this workflow trigger is  `pull_request against main branch only` which means that checkstyle reporter will not work on downstream forks.

I have proposed a fix for it on a different PR https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/pull/1595. Please take a look either we can merge PR #1595 quickly or we can incorporate changes in your PR. 
Thank you.

Reply:
let's keep them separated, I thought that it should have been enough